warehouse/query-core/src/main/java/datawave/query/cardinality/CardinalityRecord.java

```diff
@@ -41,7 +41,7 @@ public CardinalityRecord(Set<String> recordedFields, DateType dateType) {
     }
     
     public CardinalityRecord(CardinalityRecord other) {
-        synchronized (cardinalityMap) {
```

Comment:
[ERROR] Synchronization on CardinalityRecord.cardinalityMap in futile attempt to guard it [datawave.query.cardinality.CardinalityRecord] At CardinalityRecord.java:[line 49] ML_SYNC_ON_FIELD_TO_GUARD_CHANGING_THAT_FIELD